### PR TITLE
Added filter for phpMyAdmin+syslog

### LINKED ISSUE #1713: phpMyAdmin has syslog output of failed authentication attempts
> Log failed authentication attempts to syslog (useful for parsing by fail2ban, for example)

https://www.phpmyadmin.net/news/2017/3/8/phpmyadmin-470-rc-released/

We may add a filter for it.

### PR DESCRIPTION
**Pull request by kmzby**
Added a minor filter for phpMyAdmin>=4.7.0 which now has an ability to log auth-related events via Syslog.
Closes #1713 

P.S. Pretty much unsure about ChangeLog, sorry.

### CODE REVIEW

**Comment by kmzby** [comment]
Sorry for spamming with commits, I'm really new to github and making PRs :(

**Comment by kmzby** [comment]
I don't really get why test fails on timestamp comparison. Is there some doc stating that 'time' in failJSON has to be 1093179022.0?
Thanks in advance.

File: fail2ban/tests/files/logs/phpmyadmin-syslog.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# failJSON: { "time": "2017-08-22T14:50:22", "match": true , "host": "81.62.21.201" }
```
Comment (sebres) [inline fail2ban/tests/files/logs/phpmyadmin-syslog.conf]: Please remove this file...

File: config/filter.d/phpmyadmin-syslog.conf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[INCLUDES]
+
+before = common.conf
+
+[Definition]
+
+_daemon = phpMyAdmin
+
+failregex = ^%(__prefix_line)suser denied: .* \(mysql-denied\) from <HOST>\s*$
```
Comment (sebres) [inline config/filter.d/phpmyadmin-syslog.conf]: I don't like greedy catch-all (especially before host), also if regex is both sides anchored.

I suspect the user is never something else as `\S+`, but for the case of bothering from script-kiddie (ddosing):
```diff
-failregex = ^%(__prefix_line)suser denied: .* \(mysql-denied\) from <HOST>\s*$
+failregex = ^%(__prefix_line)suser denied: (?:\S+|.*?) \(mysql-denied\) from <HOST>\s*$
```

File: fail2ban/tests/files/logs/phpmyadmin-syslog [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# failJSON: { "time": "2017-08-22T14:50:22", "match": true , "host": "81.62.21.201" }
```
Comment (sebres) [inline fail2ban/tests/files/logs/phpmyadmin-syslog]: BTW. Usage of real IPs is is considered bad. Suggestion: 192.0.2.1 from test-net-1.

**Comment by kmzby** [comment]
Thanks!

**Pull request merged**